exporters/logging-otlp/src/main/java/io/opentelemetry/exporter/logging/otlp/internal/OtlpStdoutExporterBuilderUtil.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.exporter.logging.otlp.internal;
+
+import io.opentelemetry.sdk.common.export.MemoryMode;
+
+public class OtlpStdoutExporterBuilderUtil {
```

Comment:
Missing internal boilerplate javadoc.

Reply:
why did the build not fail?

Reply:
There's not a build check for it right now. Do you know if `opentelemetry-java-instrumentation` managed to add this automation yet?

Reply:
Opened new issue here: https://github.com/open-telemetry/opentelemetry-java/issues/6798